### ISSUE TITLE
Community Model Repo

### ISSUE THREAD

**Issue by DavidMansolino** [cyberbotics/webots#1874]
Many users are not publishing their models/controllers because our quality assurance and quality standard are quite high.

We have plans for allowing the users to share their model, but these are long term plans. In the meantime, a simple solution would be to create a 'webots-coomunity' (or anything similar) repo where users could add their models/controllers/simulations with a lower quality standard and without too detailed review.

**Comment by Simon-Steinmann** [comment]
This would be great. Almost a year ago, I tried out webots as an alternative to Gazebo, but ultimately dropped, because I didn't manage to implement the jaco2 kinova arm I was working on. Even with using the urdf converter. 
For the future implemented feature, I attached a image, of how it could look like[
![unofficial_protos_example](https://user-images.githubusercontent.com/52213164/86474669-99dbd180-bd43-11ea-8383-ce678df9155a.png)
](url)
